### ISSUE TITLE
Mobile support in chrome browser

### ISSUE THREAD

**Issue by afzalsayed96** [papercss/papercss#103]
The mobile view in chrome browser is not showing up as expected.
Possible Reason:
1. Download section

![image](https://user-images.githubusercontent.com/14029371/33760703-92c3c11e-dc2c-11e7-86d7-b5d9bf08551a.png)

2. Non-responsive image section

![image](https://user-images.githubusercontent.com/14029371/33760791-f851f9ec-dc2c-11e7-9897-e9f16d0bf9a8.png)

**Comment by TotomInc** [comment]
Looks like we need to add an overflow on the container for the text. About the image, it should be a max-width IMO.

**Comment by afzalsayed96** [comment]
Mobile view in firefox is perfect. For chrome I think the following code will be required to fix this
```
.section {
  word-wrap: break-word;
}
```

```
img.no-responsive {
    max-width: -moz-available;           //firefox
    max-width: -webkit-fill-available; //chrome
}
```

**Comment by afzalsayed96** [comment]
Review the fix in PR #107

**Comment by TotomInc** [comment]
#107 have been merged. Thanks for the fix @afzalsayed96!
